content/_partners/contact._en.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
All of these content page should include a `title:` in their YAML frontmatter metadata. Otherwise, the tab title is inherited from the file name:

![image](https://user-images.githubusercontent.com/1779930/161975955-d01307ed-0777-441a-ba08-610d748fd32c.png)

Reply:
This still hasn't been addressed.

(Not a blocker for initial merge)

Reply:
Filed as [LG-6407](https://cm-jira.usa.gov/browse/LG-6407)